superset/assets/src/SqlLab/components/SqlEditorLeftBar.jsx

```diff
@@ -196,7 +195,7 @@ class SqlEditorLeftBar extends React.PureComponent {
               <RefreshLabel
                 onClick={this.onDatabaseChange.bind(
                     this, { value: database.id }, true)}
-                tooltipContent="force refresh schema list"
+                tooltipContent="force refresh table list"
```

Comment:
shouldn't this say `schema`?

Reply:
Also, don't forget `i18n` as in `tooltipContent={t('force refresh table list')}`

Reply:
woops. my bad.